dotcom-rendering/cypress/e2e/parallel-1/sign-in-gate.cy.js

```diff
@@ -489,7 +489,7 @@ describe('Sign In Gate Tests', function () {
 					);
 					cy.get('[data-cy=sign-in-gate-main_register]')
 						.should('have.attr', 'href')
-						.and('contains', '/signin?returnUrl=')
+						.and('contains', '/register?returnUrl=')
 						.and(
```

Comment:
It might be worth updating the test here: 
https://github.com/guardian/dotcom-rendering/blob/842c602d70eff2ce86c2861f8258def6fab833aa/dotcom-rendering/cypress/e2e/parallel-1/sign-in-gate.cy.js#L189 
and https://github.com/guardian/dotcom-rendering/blob/842c602d70eff2ce86c2861f8258def6fab833aa/dotcom-rendering/cypress/e2e/parallel-1/sign-in-gate.cy.js#L199
to be explicit about which url the button is linking to

Reply:
good catch. thanks.